runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -231,6 +231,11 @@ public void popupMenuCanceled(PopupMenuEvent popupMenuEvent)
 		add(container, BorderLayout.NORTH);
 	}
 
+	void sortByXp(int position)
```

Comment:
This isn't a very good method name since it doesn't sort the box, it simply sets its z order. Change the method name accordingly